### PR TITLE
#169005624 Create signup database endpoint

### PR DESCRIPTION
**Pull request by Skemc**
## What does this PR do?
- Set up database signup for my app
## Description of Task to be completed?
- connect database with my app on signup
- add test

## How should this be manually tested?
- Clone this repo
- start the server by typing **npm start**
- enter the required fields and it should respond with a 200 HTTP status code, otherwise it should respond with a relevant error message
- follow UI pages
## Any background context you want to provide?
- N/A
## What are the relevant pivotal tracker stories?
[#169005624](https://www.pivotaltracker.com/story/show/169005624)
## Screenshot (if applicable)
image

### CODE REVIEW

File: v2/server/controllers/article.controller.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
+import users from '../models/user.model';
+import articles from '../models/article.model';
+import validateArticles from '../helpers/article.validations';
+import comments from '../models/comment.model';
+ 
+class ArticleController{
```
Comment (ekumamait) [inline v2/server/controllers/article.controller.js]: This should be in a different Pull Request

File: v2/server/helpers/user.validation.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import Joi  from "@hapi/joi";
+const names = /^[a-zA-Z]{2,}$/
+const email = /^\S+@[\w\-]+\.[A-Za-z ]{2,}$/;
+const password = /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.{8,})/;
```
Comment (ekumamait) [inline v2/server/helpers/user.validation.js]: Where are you using these constants you have declared?

**Review by ekumamait — CHANGES_REQUESTED** [review]
I want to understand why the articles Controller is in this Pull Request

**Comment by Skemc** [comment]
****
